services/dynamic-sidecar/openapi.json

```diff
@@ -224,9 +224,9 @@
         "tags": [
           "containers"
         ],
-        "summary": "Get Entrypoint Container Name",
```

Comment:
Entrypoint container was probably referring to the master container in the stack?

Reply:
yes